masternodes/setup-testnet.rst

```diff
@@ -495,11 +486,11 @@ You can check the status of your masternode using the various ``dashmate
 status`` commands as follows::
 
 - dashmate status
-- dashmate status:core
-- dashmate status:host
-- dashmate status:masternode
-- dashmate status:platform
-- dashmate status:services
+- dashmate status core
+- dashmate status host
+- dashmate status masternode
+- dashmate status platform
+- dashmate status services
 
 Continue with the :ref:`Registration step <testnet-register-masternode>` to
 setup the collateral, keys and construct the ProTx transaction required
```

Comment:
It's not possible to add a comment to specific line which is not modified. 
Since we install dashmate by npm there's no point to keep 
`git checkout master
  git pull
  npm ci`
We definitely should commands like:
`npm update -g dashmate` # update dashmate itself
`dashmate update` # use latest platform component versions (Drive, DAPI)

Reply:
Good catch, this has been fixed